app/components/nav-modal-binding.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-frost-navigation/components/nav-modal-binding'
```

Comment:
Can't comment on the image, but this is for the image - the opacity may be a bit too much, I don't think we want to see that much of the background.  Maybe closer to a .9?